tests/ui/implied-bounds/auxiliary/bevy_ecs.rs

```diff
@@ -0,0 +1,20 @@
+// Related to Bevy regression #118553
+
+pub mod system {
```

Comment:
Please just online this into the test itself - the special casing does not use the path.